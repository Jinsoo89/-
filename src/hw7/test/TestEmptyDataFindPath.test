# Test Dijkstra's algorithm when there is no data

LoadGraph empty emptyData.tsv

ListNodes empty

FindPath empty n1 n2