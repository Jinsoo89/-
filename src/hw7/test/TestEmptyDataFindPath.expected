# Test Dijkstra's algorithm when there is no data

loaded graph empty

empty contains:

unknown character n1
unknown character n2